clippy_workspace_tests/build.rs

```diff
@@ -0,0 +1,5 @@
+fn main() {
```

Comment:
```suggestion
#![deny(clippy::print_stdout)]

fn main() {
```

Reply:
Given that `print_stdout` is a restriction lint we have to enable it manually. I think that may be the reason it didn't work for you.

Another thing to take into account is that cargo will not re-run the build script if the script did not change since the last build.

Reply:
Thanks! As you say, `print_stdout` lint needed to be manually enabled.